release-manager/files/README.md

```diff
@@ -60,17 +60,31 @@ repositories:
 
 This type designates ODS components designed for _code development_. Such repositories are based on quickstarters whose names start with `be-`, `ds-`, or `fe-`, for _backend_, _data science_, and _frontend_, respectively. This is the default type.
 
+If you use this type ODS expects to find JUnit XML test results. If you do not have any test results the pipeline will fail. If you are deploying something where JUnit XML test results are not available consider using <<Repository Type: ods-service>>.
+
+#### Repository Type: ods-infra
+
+This type designates ODS components designed for _consuming on-prem or cloud services_ of arbitrary type using infrastructure as code. Such components are based on quickstarters whose names start with `infra-`.
```

Comment:
I guess the prefix is `inf-`.

Also which expectations does release manager create for this type?
It looks like some reports / folder has to be available.

Reply:
the prefix mentioned in the document is wrong as you point out.